action.yml

```
@@ -72,7 +71,9 @@ inputs:
     required: false
     default: "github-actions[bot]@users.noreply.github.com"
   sign-commits:
-    description: "Set to true if the action should sign the commit with GPG"
+    description: "Set to true if the action should sign the commit"
```

```suggestion
    description: "Set to true if the action should sign the commit with the GitHub Actions token"
```

or something, to make it more explicit that it should only be set in case you want the commit signed by the token and not by GPG, etc